src/main/java/EntryPoint/dto/ResponseDTO.java

```diff
@@ -0,0 +1,31 @@
+package EntryPoint.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
+
+@Data
+@AllArgsConstructor
+public class ResponseDTO {
```

Comment:
Please rename the class name as ApiResponseDTO.

Reply:
resolved